src/components/MyVisitForm/MyVisitForm.jsx

```diff
@@ -102,46 +107,75 @@ export const MyVisitForm = () => {
     setDoctorId(selectedDoctor);
   }, [doctor, users]);
 
+  useEffect(() => {
+    let breedName = '';
+    usersPets.forEach((el) => {
+      if (pet === el.name) {
+        breedName = el.breed;
+      }
+    });
+    setBreed(breedName);
+  }, [pet, usersPets]);
+
+  const [enteredPetTouched, setEnteredPetTouched] = useState(false);
+  const [enteredDoctorTouched, setEnteredDoctorTouched] = useState(false);
+  const enteredPetIsValid = pet !== '';
+  const enteredDoctorIsValid = doctor !== '';
```

Comment:
you use this values inside handlesubmit function, please move them there

Reply:
I had a double-click issue when I added those values inside handlesubmit function, so instead I used useState to manage them. Hope that's also ok...

Reply:
perfect!